gd32c10x/standard_peripheral/Source/gd32c10x_can.c

```diff
@@ -36,6 +36,15 @@ OF SUCH DAMAGE.
 #include "gd32c10x_can.h"
 #include <stdlib.h>
 
+/* CAN communication mode */
+#define CAN_NORMAL_MODE                    ((uint8_t)0x00U)             /*!< normal communication mode */
+#define CAN_LOOPBACK_MODE                  ((uint8_t)0x01U)             /*!< loopback communication mode */
+#define CAN_SILENT_MODE                    ((uint8_t)0x02U)             /*!< silent communication mode */
+#define CAN_SILENT_LOOPBACK_MODE           ((uint8_t)0x03U)             /*!< loopback and silent communication mode */
+
+/* CAN timeout */
+#define CAN_TIMEOUT                        ((uint32_t)0x0000FFFFU)      /*!< timeout value */
```

Comment:
this definitions need to be public, see https://github.com/zephyrproject-rtos/hal_gigadevice/blob/main/gd32f4xx/standard_peripheral/include/gd32f4xx_can.h#L597